plugin/settings/settings.css

```diff
@@ -311,3 +309,87 @@
   margin-bottom:15px;
   border: 1px solid #c3c4c7;
 }
+
+.tangible-plugin-features-cards {
+  display: flex;
+  gap: 0.5rem;
+  flex-wrap: wrap;
+}
+
+.tangible-card {
+  padding: 15px;
+  background-color: #fff;
+  border: 1px solid #ddd;
+  border-radius: 5px;
+  display: flex; 
+  flex-direction: column; 
+  height: 7rem;
+}
+
+.tangible-card .feature-description{
+  overflow-y: auto;
+}
+
+.card-content {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.tangible-plugin-features-cards .setting-row{
+  max-width: 20rem;
+  flex: 1 1 100%;
+
+  @media only screen and (max-width: 697px) {
+      max-width: 100%;
+  }
+}
+
+.tangible-feature-label {
+  font-weight: 600;
+}
+
+.tangible-feature-switch {
+  display: flex;
+  cursor: pointer;
+  justify-content: space-between;
+}
+
+.tangible-feature-switch input[type="checkbox"] {
+  display: none; 
+}
+
+.tangible-feature-switch-label{
+  font-weight: 600;
+  font-size: .9rem;
+}
+
+.tangible-slider {
```

Comment:
I would name this CSS class differently, `slider` is a bit ambiguous. Maybe `switch-slider`?

There's a name for this specific UI element, but I don't recall. I think I saw it in the [Open UI project](https://open-ui.org).

> The purpose of the Open UI, a [W3C Community Group](https://www.w3.org/community/open-ui/), is to allow web developers to style and extend built-in web UI components and controls, such as <select> dropdowns, checkboxes, radio buttons, and date/color pickers.
>
> To do that, we’ll need to fully specify the **component parts, states, and behaviors** of the built-in controls, as well as necessary **accessibility** requirements.

Here's the Switch component.

- https://open-ui.org/components/switch/

Perfect, it's a good overview, including:

#### Anatonmy

- checkedContent
- uncheckedContent
- root
- thumb
- loadingContent
- track

There's no description of each element. Looks like "track" is the word we want.